docker/hoodie/hadoop/hive_base/README.md

```diff
@@ -0,0 +1,49 @@
+[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/big-data-europe/Lobby)
+
+# docker-hive
```

Comment:
@bvaradar is this file meant to be checked in?

Reply:
Missed this. I have removed it now.